bulbs/page/enum.py

```diff
@@ -0,0 +1,14 @@
+from django_enumfield import enum
+
+
+class Status(enum.Enum):
+    DRAFT = 0
+    PUBLISHED = 1
+
+
+class TemplateType(enum.Enum):
```

Comment:
@spra85 I just got these values from the mocks. Are there any other template types that we know of?